### DISCUSSION TITLE
DFU on Jumper T-Lite V2

### DISCUSSION THREAD

**Discussion by Shlipez** [ExpressLRS/ExpressLRS#3062] [Q&A]
Please help me, I don't understand how to turn on the DFU on Jumper T-Lite V2, it says on the Internet to turn off the controller, hold down some horizontal buttons, press the on button and turn off the controller, I tried to do this, nothing worked, although a sign appeared on the screen with the inscription: ,,Key stuck."

**Comment by Mkorho** [comment]
https://www.expresslrs.org/quick-start/transmitters/jumper-internal/#via-etx-passthrough
